sdks/python/apache_beam/runners/interactive/interactive_beam.py

```diff
@@ -339,151 +342,229 @@ def record(self, pipeline):
 
 
 class Clusters:
-  """An interface for users to modify the pipelines that are being run by the
-  Interactive Environment.
+  """An interface to manage clusters running workers that are connected with
```

Comment:
This file is starting to get pretty big. It doesn't have to be in this PR, but we may want to start investigating moving some of this logic to sub-modules.